tests/storage/restore.go

```diff
@@ -1946,39 +1881,17 @@ var _ = Describe(SIG("VirtualMachineRestore Tests", func() {
 			})
 
 			It("should restore with volume restore policy InPlace and PVC as disk", func() {
-				// VM with normal DV mounted to it
-				vm = createVMWithCloudInit(cd.ContainerDiskCirros, snapshotStorageClass)
-				vm.Spec.DataVolumeTemplates = nil // Remove traces of DV, we want a raw PVC
-				vm.Spec.Template.Spec.Volumes = vm.Spec.Template.Spec.Volumes[1:]
-
-				// Create and mount PVC to VM
 				pvcName := "standalone-pvc"
-				pvc := &corev1.PersistentVolumeClaim{
-					ObjectMeta: metav1.ObjectMeta{
-						Name:      pvcName,
-						Namespace: vm.Namespace,
-					},
-					Spec: corev1.PersistentVolumeClaimSpec{
-						AccessModes: []corev1.PersistentVolumeAccessMode{corev1.ReadWriteOnce},
-						Resources: corev1.VolumeResourceRequirements{
-							Requests: corev1.ResourceList{
-								"storage": resource.MustParse("2Gi"),
-							},
-						},
-					},
-				}
 
-				vm.Spec.Template.Spec.Volumes = append([]v1.Volume{{
-					Name: "disk0",
-					VolumeSource: v1.VolumeSource{
-						PersistentVolumeClaim: &v1.PersistentVolumeClaimVolumeSource{
-							PersistentVolumeClaimVolumeSource: corev1.PersistentVolumeClaimVolumeSource{
-								ClaimName: pvcName,
-							},
-						},
-					},
-				}}, vm.Spec.Template.Spec.Volumes...)
+				vm = libvmi.NewVirtualMachine(
```

Comment:
mind moving the vm definition to after the pvc creation? since you use the pvc name in the vm definition